src/easier68k/core/enum/op_size.py

```diff
@@ -20,6 +20,15 @@ def parse(code: chr) -> int:
             return MoveSize.WORD
         if code == 'L':
             return MoveSize.LONG
+    
+    @staticmethod
+    def parse_binary(code: int) -> chr:
```

Comment:
should this be made to be a util method instead

Reply:
Done

Reply:
Oh I miss read this. But I don't think so. Not unless we also move the parse function (they are inverses of each other)